tox.ini

```diff
@@ -17,9 +17,10 @@ deps =
     # Python3.5+ only deps
     py{35,36}: aiohttp >= 2.3.0
     py{35,36}: pytest-aiohttp
+    py{35,36}: aiobotocore
 
 commands =
-    py{27,34}: coverage run --source aws_xray_sdk -m py.test tests --ignore tests/ext/aiohttp --ignore tests/test_async_local_storage.py
+    py{27,34}: coverage run --source aws_xray_sdk -m py.test tests --ignore tests/ext/aiohttp --ignore tests/ext/aiobotocore --ignore tests/test_async_local_storage.py --ignore tests/test_async_recorder.py
```

Comment:
Again the keyword you used for capture_async actually works with 3.4 but the aiobotocore patcher only work with 3.5.

Reply:
Have used async/await throughout so best to exclude those tests from py34

Reply:
Yeah I mean let's get all async related stuff to be 3.5+ only.

Reply:
Yeah :smile:, async in 3.4 is pretty nasty anyway. I "think" I've excluded 3.4 from anything async and purged any `@asyncio.coroutine`'s too